src/release.js

```diff
@@ -165,11 +168,17 @@ export default function taoExtensionReleaseFactory(params = {}) {
         },
 
         /**
-         * Publish the released package
+         * Publish the released package(s)
          * @returns {Promise}
          */
         async publish() {
-            return await adaptee.publish();
+            if (params.publish) {
+                if (data.monorepoPackages) {
+                    return await adaptee.monorepoPublish();
+                } else {
+                    return await adaptee.publish();
```

Comment:
I feel like we can skip `await` here, and just return the promise from the `adaptee` function.
I mean, the code invoking `release.publish()` will do its own `await` on top of it anyway.

Reply:
You are right.

Though I actually like this syntax with additional `await`, because it's clear that the funciton is `async`. If you remove `await`, it will only remain in jsdoc.